.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,49 @@
+name: Feature request
+description: Suggest a new feature or enhancement
+title: "[FEATURE] "
+labels: [enhancement]
```

Comment:
name: Feature Request 🧑‍💻
title: [FEATURE 🧑‍💻]: '
labels: ["Up-for-Grabs ✋", "enhancement 🧑‍💻"]